presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -270,24 +270,20 @@ public AggregationBuilder addAggregation(Symbol output, Aggregation aggregation)
 
         public AggregationBuilder globalGrouping()
         {
-            return groupingSets(ImmutableList.of(ImmutableList.of()));
-        }
-
-        public AggregationBuilder groupingSets(List<List<Symbol>> groupingSets)
-        {
-            checkState(this.groupingSets.isEmpty(), "groupingSets already defined");
-            this.groupingSets.addAll(groupingSets);
+            groupingSets(AggregationNode.singleGroupingSet(ImmutableList.of()));
             return this;
         }
 
-        public AggregationBuilder addGroupingSet(Symbol... symbols)
+        public AggregationBuilder singleGroupingSet(Symbol... symbols)
```

Comment:
Not a huge fan of builder methods without add, also the function name could be confusing because of conflict with util methods in AggregatioNode

Reply:
It's not additive, though. You couldn't call `addSingleGroupingSet` multiple times and expect it to work.